README.md

```diff
@@ -19,56 +19,100 @@ bin/check <files>
 ```
 » bin/check test/fixtures/mods/*
 
+* test/fixtures/mods/cusbmss228-p00001-invalid.xml:
+  Value of dateCreated: 'long ago' is not W3C-valid (https://www.w3.org/TR/1998/NOTE-datetime-19980827).
+
 * test/fixtures/mods/cusbmss228-p00001-latin1.xml:
   Missing 'encoding="UTF-8"' declaration.
 
 * test/fixtures/mods/cusbmss228-p00001-missing-declaration.xml:
   Missing 'encoding="UTF-8"' declaration.
 
+* test/fixtures/mods/cusbmss228-p00001-uncontrolled.xml:
+  12:0: ERROR: Element '{http://www.loc.gov/mods/v3}typeOfResource': [facet 'enumeration'] The value 'wiggly image' is not an element of the set {'text', 'cartographic', 'notated music', 'sound recording-musical', 'sound recording-nonmusical', 'sound recording', 'still image', 'moving image', 'three dimensional object', 'software, multimedia', 'mixed material', ''}.
+
+* test/fixtures/mods/cusbmss228-p00001-uncontrolled.xml:
+  12:0: ERROR: Element '{http://www.loc.gov/mods/v3}typeOfResource': 'wiggly image' is not a valid value of the atomic type '{http://www.loc.gov/mods/v3}resourceTypeDefinition'.
```

Comment:
@dunn: I am a bit confused by your comment above. MODS resourceType is not the type that was referenced in the ticket (that type refers to the Fedora object type).MODS resourceType maps to work_type in the CSV files. We don't explicitly state the Fedora object types anywhere in the MODS.